### DISCUSSION TITLE
Issue trying to install zksync plugin

### DISCUSSION THREAD

**Discussion by GALDUNX** [Cyfrin/foundry-full-course-cu#2576] [General]
i tried installing zksync plugin but i am unable to install .. After typing Zksync in the required search bar(Zksync doesnt pup up) then  i see a button that say permission and when i click on it ..it say no permission requested yet but before that i see an underline text saying connect to local plugin and then i am lost.. please can you guide me on what to do so i can be able to install zksync plugin
<img width="1440" alt="Screenshot 2024-09-29 at 18 23 45" src="https://github.com/user-attachments/assets/68a04d1b-625d-42ac-b396-d239013c6ae9">
<img width="1440" alt="Screenshot 2024-09-29 at 18 23 50" src="https://github.com/user-attachments/assets/08c3f166-1283-4060-bfa8-90258af10de1">

**Comment by EngrPips** [comment]
It worked for me, maybe reset your browser setting then retry or try another browser.

<img width="1440" alt="Screenshot 2024-09-29 at 20 51 10" src="https://github.com/user-attachments/assets/942c5678-8890-4d3a-acf5-f2b93289ebdf">

**Reply by GALDUNX** [reply]
ok thanks very much

**Reply by EngrPips** [reply]
Anytime, Friend.

**Comment by chainflash** [comment]


**Reply by EngrPips** [reply]
Interesting
